frontstage/logger_config.py

```diff
@@ -20,9 +20,9 @@ def add_service(logger, method_name, event_dict):  # pylint: disable=unused-argu
         return event_dict
 
     logging.basicConfig(stream=sys.stdout, level=log_level, format=logger_format)
-    oauth_log = logging.getLogger("requests_oauthlib")
-    oauth_log.addHandler(logging.NullHandler())
-    oauth_log.propagate = False
+    auth_log = logging.getLogger("requests_authlib")
```

Comment:
This whole bit might be able to go.  requests_oauthlib is the name of a python library that was removed recently when we had a bit of a tidyup of the libraries.  I think this block was handling the logging for that library which isn't here anymore.

Reply:
fixed